runtime/src/configs/mod.rs

```diff
@@ -420,6 +427,13 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
     type WeightInfo = weights::cumulus_pallet_xcmp_queue::WeightInfo<Runtime>;
     // Enqueue XCMP messages from siblings for later processing.
     type XcmpQueue = TransformOrigin<MessageQueue, AggregateMessageOrigin, ParaId, ParaIdToSibling>;
+    type MaxActiveOutboundChannels = ConstU32<128>;
+	type MaxPageSize = ConstU32<{ 1 << 16 }>;
```

Comment:
On-chain HRMP channels typically have a maximum page size of `100 * 1024` bytes. However, 2^16 = 65536 is significantly lower. I recommend setting this value to `103 * 1024` bytes.